test/storage-local/datasets.test.ts

```diff
@@ -0,0 +1,343 @@
+import { emptyDirSync, ensureDirSync, readdirSync, readFile, writeJsonSync } from 'fs-extra';
+import { join } from 'path';
+import { ApifyStorageLocal } from '@apify/storage-local';
+import { STORAGE_NAMES } from '@apify/storage-local/src/consts';
```

Comment:
dont use deep imports

```suggestion
import { ApifyStorageLocal, STORAGE_NAMES } from '@apify/storage-local';
```

if the symbol is not exported, export it